sdk/spring/azure-spring-boot-samples/azure-cloud-foundry-service-sample/pom.xml

```diff
@@ -22,7 +22,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-cosmosdb-spring-boot-starter</artifactId>
-      <version>2.3.3</version> <!-- {x-version-update;com.microsoft.azure:azure-cosmosdb-spring-boot-starter;dependency} -->
+      <version>2.3.3</version> <!-- {x-version-update;com.microsoft.azure:azure-cosmosdb-spring-boot-starter;external_dependency} -->
```

Comment:
In sample projects, please use `current` .

Related PR: https://github.com/Azure/azure-sdk-for-java/pull/14507